docs/rules.rst

```diff
@@ -0,0 +1,25 @@
+=====
+Rules
+=====
+
+Event driven automation uses rules to determine if an action should be taken when an event
+is received.  The rule decides to run an action by using a condition that is supplied
+by the user.   This condition can use information from the event received, information
+from recently previous events, longer term facts about the system, or variables provided
+by the user.
+
+The following is an example rule::
+
+    name:  An automatic remedation rule
+    condition:  event.outage == True
+    action:
+        run_playbook:
+            name: remediate_outage.yml
+
+This rule searches for a recent event with the data `outage` being `True`. If an event
+with this condition is found then the `remediate_outage.yml` playbook is run.
+
+
```

Comment:
@benthomasson What are the constraints when multiple rules are specified in a single file. Can the events be reused across rules. Do rule names have to be unique within a file?

Reply:
Each rule set is a separate namespace.    An event only matches one rule.    If we want to match multiple rules we need to promote it to a fact.

Reply:
Rule names are just documentation.   It is probably a good idea if they are unique, but I don't enforce that anywhere.

Reply:
If a rule names is not provided durable rules adds a default name of r_{index} e.g r_0